src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -759,6 +765,8 @@ namespace DataHeatBalance {
         void SetWindSpeedAt(EnergyPlusData &state, Real64 fac);
 
         void SetWindDirAt(Real64 fac);
+
+        Real64 sumHATsurf(EnergyPlusData &state);
```

Comment:
New method for zone and space to replace 8 duplicate sumHAT functions in places like radiant systems.